moveit_setup_assistant/CMakeLists.txt

```diff
@@ -1,169 +1,91 @@
-else()
-  find_package(Qt5 ${rviz_QT_VERSION} REQUIRED Core Widgets)
-  set(QT_LIBRARIES Qt5::Widgets)
-endif()
-# Instruct CMake to run moc automatically when needed.
-set(CMAKE_AUTOMOC ON)
-add_definitions(-DQT_NO_KEYWORDS)
 
-# Support new yaml-cpp API.
-find_package(PkgConfig)
-pkg_check_modules(YAMLCPP REQUIRED yaml-cpp>=0.5)
-include_directories(SYSTEM ${YAMLCPP_INCLUDE_DIRS})
-
-catkin_package(
-  INCLUDE_DIRS
-    include
-  LIBRARIES
-    ${PROJECT_NAME}_tools
-  CATKIN_DEPENDS
-    moveit_ros_planning
-    moveit_ros_visualization
-    roscpp
-)
-
-# Header files that need Qt Moc pre-processing for use with Qt signals, etc:
-set(HEADERS
-  src/widgets/navigation_widget.h
-  src/widgets/header_widget.h
-  src/widgets/setup_assistant_widget.h
-  src/widgets/start_screen_widget.h
-  src/widgets/planning_groups_widget.h
-  src/widgets/double_list_widget.h
-  src/widgets/kinematic_chain_widget.h
-  src/widgets/group_edit_widget.h
-  src/widgets/default_collisions_widget.h
-  src/widgets/robot_poses_widget.h
-  src/widgets/end_effectors_widget.h
-  src/widgets/virtual_joints_widget.h
-  src/widgets/passive_joints_widget.h
-  src/widgets/perception_widget.h
-  src/widgets/ros_controllers_widget.h
-  src/widgets/controller_edit_widget.h
-  src/widgets/configuration_files_widget.h
-  src/widgets/setup_screen_widget.h
-  src/widgets/simulation_widget.h
-  src/widgets/author_information_widget.h
-)
 
 # Tools Library
 add_library(${PROJECT_NAME}_tools
   src/tools/compute_default_collisions.cpp
   src/tools/moveit_config_data.cpp
-  src/tools/collision_linear_model.cpp
-  src/tools/collision_matrix_model.cpp
-  src/tools/rotated_header_view.cpp
 )
-set_target_properties(${PROJECT_NAME}_tools PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
-target_link_libraries(${PROJECT_NAME}_tools
-  ${YAMLCPP_LIBRARIES}
-  ${catkin_LIBRARIES}
-  ${Boost_LIBRARIES}
-  ${QT_LIBRARIES}
+target_include_directories(${PROJECT_NAME}_tools PUBLIC
+  $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+  $<INSTALL_INTERFACE:include>
+  ${OMPL_INCLUDE_DIRS}
 )
-
-# Main Widgets Library - all screens (navigation options)
-add_library(${PROJECT_NAME}_widgets
-  src/widgets/start_screen_widget.cpp
-  src/widgets/planning_groups_widget.cpp
-  src/widgets/double_list_widget.cpp
-  src/widgets/kinematic_chain_widget.cpp
-  src/widgets/group_edit_widget.cpp
-  src/widgets/default_collisions_widget.cpp
-  src/widgets/robot_poses_widget.cpp
-  src/widgets/end_effectors_widget.cpp
-  src/widgets/virtual_joints_widget.cpp
-  src/widgets/passive_joints_widget.cpp
-  src/widgets/perception_widget.cpp
-  src/widgets/configuration_files_widget.cpp
-  src/widgets/ros_controllers_widget.cpp
-  src/widgets/controller_edit_widget.cpp
-  src/widgets/navigation_widget.cpp
-  src/widgets/header_widget.cpp
-  src/widgets/setup_assistant_widget.cpp
-  src/widgets/setup_screen_widget.cpp
-  src/widgets/simulation_widget.cpp
-  src/widgets/author_information_widget.cpp
-  ${HEADERS}
```

Comment:
Comment out these lines, don't delete them, we still want to port them after your PR